app/Console/Commands/GenerateArticle.php

```diff
@@ -39,15 +39,12 @@ class GenerateArticle extends Command
     protected $jsonEncodedUserInput;
 
     /**
-     * @var Illuminate\Filesystem\Filesystem
+     * Variable to store filesystem object
+     *
+     * @var Filesystem
```

Comment:
Filesystem is not in `use` statement. So it make sense to give full namespace.

`Illuminate\Filesystem\Filesystem`